applications/openshift/api-server/audit_log_forwarding_is_enabled/rule.yml

```diff
@@ -0,0 +1,32 @@
+prodtype: ocp4
```

Comment:
s/audit_log_forwarding_is_enabled/audit_log_forward_enabled/g 

Don't need verbs like `is, are, etc.` in rule names.